MDANSE/Src/MDANSE/Trajectory/MdanseTrajectory.py

```diff
@@ -330,7 +330,7 @@ def to_real_coordinates(self, box_coordinates, first, last, step):
             real_coordinates = np.empty(box_coordinates.shape, dtype=np.float64)
             comp = 0
             for i in range(first, last, step):
-                direct_cell = self._unit_cells[i].transposed_direct
+                direct_cell = self._unit_cells[i].direct
                 real_coordinates[comp, :] = np.matmul(
                     direct_cell, box_coordinates[comp, :]
                 )
```

Comment:
`@` is a shortcut for `matmul`

> The matmul function implements the semantics of the @ operator introduced in Python 3.5 following [PEP 465](https://peps.python.org/pep-0465/).

```suggestion
                real_coordinates[comp, :] = direct_cell @ box_coordinates[comp, :]
```